docs/src/main/paradox/grpc-replicated-event-sourcing-transport.md

```diff
@@ -0,0 +1,146 @@
+# Akka Replicated Event Sourcing over gRPC
+
+Akka Replicated Event Sourcing extends Akka Persistence allowing multiple replicas of the same entity, all accepting
+writes, for example in different data centers or cloud provider regions. This makes it possible to implement patterns 
+such as active-active and hot standby. 
+
+Originally, Akka Replicated Event Sourcing has required cross-replica access to the underlying replica database, which
+can be hard to open up for security and infrastructure reasons.
+
+Akka Replicated Event Sourcing over gRPC builds on @ref:[Akka Projection gRPC](grpc.md) and @extref:[Akka gRPC](akka-grpc:index.html) to instead use gRPC as the cross-replica transport for events.
+
+@@@ warning
+
+This module is currently marked as [May Change](https://doc.akka.io/docs/akka/current/common/may-change.html)
+in the sense that the API might be changed based on feedback from initial usage.
+However, the module is ready for usage in production and we will not break serialization format of
+messages or stored data.
+
+@@@
+
+## Overview
+
+For a basic overview of Replicated Event Sourcing see the @extref:[Akka Replicated Event Sourcing docs](akka:typed/replicated-eventsourcing.html)
+
+Akka Replicated Event Sourcing over gRPC consists of the following three parts:
+
+The Replicated Event Sourced Behavior is run in each replica as a sharded entity using @extref:[Akka Cluster Sharding](akka:typed/cluster-sharding.html).
+
+The events of each replica are published to the other replicas using @ref:[Akka Projection gRPC](grpc.md) endpoints.
+
+Each replica consumes a number of parallel slices of the events from each other replica by running Akka Projection gRPC
+in @extref:[Akka Sharded Daemon Process](akka:typed/cluster-sharded-daemon-process.html).
+
+
+
+## Dependencies
+
+The functionality is provided through the `akka-projection-grpc` module.
+
+@@project-info{ projectId="akka-projection-grpc" }
+
+To use the gRPC module of Akka Projections add the following dependency in your project:
+
+Akka Replicated Event Sourcing over gRPC requires Akka 2.8.0 or later and can only be run in an Akka cluster since it uses cluster components.
+
+It is currently only possible to use @extref:[akka-persistence-r2dbc](akka-persistence-r2dbc:projection.html) as the
+projection storage and journal for this module.
+
+The full set of dependencies needed:
+
+@@dependency [sbt,Maven,Gradle] {
+group=com.lightbend.akka
+artifact=akka-projection-grpc_$scala.binary.version$
+version=$project.version$
+group2=com.typesafe.akka
+artifact2=akka-cluster-typed_$scala.binary.version$
+version2=2.8.0
+group3=com.typesafe.akka
+artifact3=akka-cluster-sharding-typed_$scala.binary.version$
+version3=2.8.0
+group4=com.lightbend.akka
+artifact4=akka-persistence-r2dbc_$scala.binary.version$
+version4=1.0.1
```

Comment:
should be possible to grab version from build (maybe with additional paradox properties)

Reply:
Fixed in https://github.com/akka/akka-projection/pull/757/commits/8f4e6371dd44c95206712d2a5098f3ab9535071a